VERSION

```diff
@@ -0,0 +1 @@
+0.2.3
```

Comment:
@fabxc I have added a newline to 'Version' and '.promu.yml' (Same as github.com/prometheus/prometheus/.promu.yml).